bug: The latest develop branch docker build is broken

```
Entrypoint application = js/application-7309a90a4d666b56f2e3.js js/application-7309a90a4d666b56f2e3.js.map
Entrypoint sdk = js/sdk.js js/sdk.js.map
Entrypoint superadmin_pages = js/superadmin_pages-7309a90a4d666b56f2e3.js js/superadmin_pages-7309a90a4d666b56f2e3.js.map
Entrypoint superadmin = js/superadmin-7309a90a4d666b56f2e3.js js/superadmin-7309a90a4d666b56f2e3.js.map
Entrypoint widget = js/widget-7309a90a4d666b56f2e3.js js/widget-7309a90a4d666b56f2e3.js.map
[0] ./app/javascript/packs/application.js 3.15 KiB {0} [built] [failed] [1 error]
[1] ./app/javascript/packs/sdk.js 3.15 KiB {1} [built] [failed] [1 error]
[2] ./app/javascript/packs/superadmin_pages.js 3.15 KiB {3} [built] [failed] [1 error]
[3] ./app/javascript/packs/superadmin.js 3.15 KiB {2} [built] [failed] [1 error]
[4] ./app/javascript/packs/widget.js 3.15 KiB {4} [built] [failed] [1 error]
ERROR in ./app/javascript/packs/application.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-env'
Require stack:
- /app/babel.config.js
- /app/node_modules/@babel/core/lib/config/files/module-types.js
- /app/node_modules/@babel/core/lib/config/files/configuration.js
- /app/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/@babel/core/lib/index.js
- /app/node_modules/babel-loader/lib/index.js
- /app/node_modules/loader-runner/lib/loadLoader.js
- /app/node_modules/loader-runner/lib/LoaderRunner.js
- /app/node_modules/webpack/lib/NormalModule.js
- /app/node_modules/webpack/lib/NormalModuleFactory.js
- /app/node_modules/webpack/lib/Compiler.js
- /app/node_modules/webpack/lib/webpack.js
- /app/node_modules/webpack-cli/bin/utils/validate-options.js
- /app/node_modules/webpack-cli/bin/utils/convert-argv.js
- /app/node_modules/webpack-cli/bin/cli.js
- /app/node_modules/webpack/bin/webpack.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (/app/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
at module.exports (/app/babel.config.js:30:9)
at readConfigJS (/app/node_modules/@babel/core/lib/config/files/configuration.js:186:15)
at readConfigJS.next (<anonymous>)
at Function.<anonymous> (/app/node_modules/@babel/core/lib/gensync-utils/async.js:16:3)
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:261:32)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at Function.errback (/app/node_modules/gensync/index.js:113:7)
at errback (/app/node_modules/@babel/core/lib/gensync-utils/async.js:60:18)
at async (/app/node_modules/gensync/index.js:188:31)
at onFirstPause (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:58:46)
at cachedFunction.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:44:11
at Array.forEach (<anonymous>)
at Function.async (/app/node_modules/gensync/index.js:43:15)
at Function.all (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at loadOneConfig (/app/node_modules/@babel/core/lib/config/files/configuration.js:106:34)
at loadOneConfig.next (<anonymous>)
at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:51)
at buildRootChain.next (<anonymous>)
at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:62)
at loadPrivatePartialConfig.next (<anonymous>)
at /app/node_modules/@babel/core/lib/config/partial.js:131:25
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:93:9
ERROR in ./app/javascript/packs/sdk.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-env'
Require stack:
- /app/babel.config.js
- /app/node_modules/@babel/core/lib/config/files/module-types.js
- /app/node_modules/@babel/core/lib/config/files/configuration.js
- /app/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/@babel/core/lib/index.js
- /app/node_modules/babel-loader/lib/index.js
- /app/node_modules/loader-runner/lib/loadLoader.js
- /app/node_modules/loader-runner/lib/LoaderRunner.js
- /app/node_modules/webpack/lib/NormalModule.js
- /app/node_modules/webpack/lib/NormalModuleFactory.js
- /app/node_modules/webpack/lib/Compiler.js
- /app/node_modules/webpack/lib/webpack.js
- /app/node_modules/webpack-cli/bin/utils/validate-options.js
- /app/node_modules/webpack-cli/bin/utils/convert-argv.js
- /app/node_modules/webpack-cli/bin/cli.js
- /app/node_modules/webpack/bin/webpack.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (/app/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
at module.exports (/app/babel.config.js:30:9)
at readConfigJS (/app/node_modules/@babel/core/lib/config/files/configuration.js:186:15)
at readConfigJS.next (<anonymous>)
at Function.<anonymous> (/app/node_modules/@babel/core/lib/gensync-utils/async.js:16:3)
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:261:32)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at Function.errback (/app/node_modules/gensync/index.js:113:7)
at errback (/app/node_modules/@babel/core/lib/gensync-utils/async.js:60:18)
at async (/app/node_modules/gensync/index.js:188:31)
at onFirstPause (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:58:46)
at cachedFunction.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:44:11
at Array.forEach (<anonymous>)
at Function.async (/app/node_modules/gensync/index.js:43:15)
at Function.all (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at loadOneConfig (/app/node_modules/@babel/core/lib/config/files/configuration.js:106:34)
at loadOneConfig.next (<anonymous>)
at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:51)
at buildRootChain.next (<anonymous>)
at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:62)
at loadPrivatePartialConfig.next (<anonymous>)
at /app/node_modules/@babel/core/lib/config/partial.js:131:25
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:93:9
ERROR in ./app/javascript/packs/superadmin_pages.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-env'
Require stack:
- /app/babel.config.js
- /app/node_modules/@babel/core/lib/config/files/module-types.js
- /app/node_modules/@babel/core/lib/config/files/configuration.js
- /app/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/@babel/core/lib/index.js
- /app/node_modules/babel-loader/lib/index.js
- /app/node_modules/loader-runner/lib/loadLoader.js
- /app/node_modules/loader-runner/lib/LoaderRunner.js
- /app/node_modules/webpack/lib/NormalModule.js
- /app/node_modules/webpack/lib/NormalModuleFactory.js
- /app/node_modules/webpack/lib/Compiler.js
- /app/node_modules/webpack/lib/webpack.js
- /app/node_modules/webpack-cli/bin/utils/validate-options.js
- /app/node_modules/webpack-cli/bin/utils/convert-argv.js
- /app/node_modules/webpack-cli/bin/cli.js
- /app/node_modules/webpack/bin/webpack.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (/app/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
at module.exports (/app/babel.config.js:30:9)
at readConfigJS (/app/node_modules/@babel/core/lib/config/files/configuration.js:186:15)
at readConfigJS.next (<anonymous>)
at Function.<anonymous> (/app/node_modules/@babel/core/lib/gensync-utils/async.js:16:3)
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:261:32)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at Function.errback (/app/node_modules/gensync/index.js:113:7)
at errback (/app/node_modules/@babel/core/lib/gensync-utils/async.js:60:18)
at async (/app/node_modules/gensync/index.js:188:31)
at onFirstPause (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:58:46)
at cachedFunction.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:44:11
at Array.forEach (<anonymous>)
at Function.async (/app/node_modules/gensync/index.js:43:15)
at Function.all (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at loadOneConfig (/app/node_modules/@babel/core/lib/config/files/configuration.js:106:34)
at loadOneConfig.next (<anonymous>)
at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:51)
at buildRootChain.next (<anonymous>)
at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:62)
at loadPrivatePartialConfig.next (<anonymous>)
at /app/node_modules/@babel/core/lib/config/partial.js:131:25
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:93:9
ERROR in ./app/javascript/packs/superadmin.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-env'
Require stack:
- /app/babel.config.js
- /app/node_modules/@babel/core/lib/config/files/module-types.js
- /app/node_modules/@babel/core/lib/config/files/configuration.js
- /app/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/@babel/core/lib/index.js
- /app/node_modules/babel-loader/lib/index.js
- /app/node_modules/loader-runner/lib/loadLoader.js
- /app/node_modules/loader-runner/lib/LoaderRunner.js
- /app/node_modules/webpack/lib/NormalModule.js
- /app/node_modules/webpack/lib/NormalModuleFactory.js
- /app/node_modules/webpack/lib/Compiler.js
- /app/node_modules/webpack/lib/webpack.js
- /app/node_modules/webpack-cli/bin/utils/validate-options.js
- /app/node_modules/webpack-cli/bin/utils/convert-argv.js
- /app/node_modules/webpack-cli/bin/cli.js
- /app/node_modules/webpack/bin/webpack.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (/app/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
at module.exports (/app/babel.config.js:30:9)
at readConfigJS (/app/node_modules/@babel/core/lib/config/files/configuration.js:186:15)
at readConfigJS.next (<anonymous>)
at Function.<anonymous> (/app/node_modules/@babel/core/lib/gensync-utils/async.js:16:3)
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:261:32)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at Function.errback (/app/node_modules/gensync/index.js:113:7)
at errback (/app/node_modules/@babel/core/lib/gensync-utils/async.js:60:18)
at async (/app/node_modules/gensync/index.js:188:31)
at onFirstPause (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:58:46)
at cachedFunction.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:44:11
at Array.forEach (<anonymous>)
at Function.async (/app/node_modules/gensync/index.js:43:15)
at Function.all (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at loadOneConfig (/app/node_modules/@babel/core/lib/config/files/configuration.js:106:34)
at loadOneConfig.next (<anonymous>)
at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:51)
at buildRootChain.next (<anonymous>)
at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:62)
at loadPrivatePartialConfig.next (<anonymous>)
at /app/node_modules/@babel/core/lib/config/partial.js:131:25
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:93:9
ERROR in ./app/javascript/packs/widget.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module '@babel/preset-env'
Require stack:
- /app/babel.config.js
- /app/node_modules/@babel/core/lib/config/files/module-types.js
- /app/node_modules/@babel/core/lib/config/files/configuration.js
- /app/node_modules/@babel/core/lib/config/files/index.js
- /app/node_modules/@babel/core/lib/index.js
- /app/node_modules/babel-loader/lib/index.js
- /app/node_modules/loader-runner/lib/loadLoader.js
- /app/node_modules/loader-runner/lib/LoaderRunner.js
- /app/node_modules/webpack/lib/NormalModule.js
- /app/node_modules/webpack/lib/NormalModuleFactory.js
- /app/node_modules/webpack/lib/Compiler.js
- /app/node_modules/webpack/lib/webpack.js
- /app/node_modules/webpack-cli/bin/utils/validate-options.js
- /app/node_modules/webpack-cli/bin/utils/convert-argv.js
- /app/node_modules/webpack-cli/bin/cli.js
- /app/node_modules/webpack/bin/webpack.js
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
at Function.Module._load (internal/modules/cjs/loader.js:725:27)
at Module.require (internal/modules/cjs/loader.js:952:19)
at require (/app/node_modules/webpack-cli/node_modules/v8-compile-cache/v8-compile-cache.js:159:20)
at module.exports (/app/babel.config.js:30:9)
at readConfigJS (/app/node_modules/@babel/core/lib/config/files/configuration.js:186:15)
at readConfigJS.next (<anonymous>)
at Function.<anonymous> (/app/node_modules/@babel/core/lib/gensync-utils/async.js:16:3)
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:261:32)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at Function.errback (/app/node_modules/gensync/index.js:113:7)
at errback (/app/node_modules/@babel/core/lib/gensync-utils/async.js:60:18)
at async (/app/node_modules/gensync/index.js:188:31)
at onFirstPause (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at cachedFunction (/app/node_modules/@babel/core/lib/config/caching.js:58:46)
at cachedFunction.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:44:11
at Array.forEach (<anonymous>)
at Function.async (/app/node_modules/gensync/index.js:43:15)
at Function.all (/app/node_modules/gensync/index.js:216:13)
at Generator.next (<anonymous>)
at loadOneConfig (/app/node_modules/@babel/core/lib/config/files/configuration.js:106:34)
at loadOneConfig.next (<anonymous>)
at buildRootChain (/app/node_modules/@babel/core/lib/config/config-chain.js:68:51)
at buildRootChain.next (<anonymous>)
at loadPrivatePartialConfig (/app/node_modules/@babel/core/lib/config/partial.js:85:62)
at loadPrivatePartialConfig.next (<anonymous>)
at /app/node_modules/@babel/core/lib/config/partial.js:131:25
at Generator.next (<anonymous>)
at step (/app/node_modules/gensync/index.js:269:25)
at evaluateAsync (/app/node_modules/gensync/index.js:291:5)
at /app/node_modules/gensync/index.js:93:9
Removing intermediate container a06b8f2555ab
The command '/bin/sh -c if [ "$RAILS_ENV" = "production" ]; then SECRET_KEY_BASE=precompile_placeholder RAILS_LOG_TO_STDOUT=enabled bundle exec rake assets:precompile; fi' returned a non-zero code: 1
```

--- a/package.json
+++ b/package.json
@@ -65,7 +65,7 @@
   "devDependencies": {
     "@babel/core": "7.13.16",
     "@babel/plugin-proposal-class-properties": "7.13.0",
-    "@babel/preset-env": "7.13.5",
+    "@babel/preset-env": "7.13.15",
     "@storybook/addon-actions": "6.2.9",
     "@storybook/addon-docs": "^6.2.9",
     "@storybook/addon-essentials": "6.2.9",

